scripts/new-release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+current=$(cat meson.build | grep -Eo "version: '(.*)'" | grep -Eo "[0-9]+\.[0-9]+.[0-9]+")
+mayor=$(echo $current | cut -d"." -f1)
+minor=$(echo $current | cut -d"." -f2)
+rev=$(echo $current | cut -d"." -f3)
+
+case $1 in
+"mayor")
+    next=$(echo $(($mayor + 1)).0.0)
+    ;;
+"minor")
+    next=$(echo $mayor.$(($minor + 1)).0)
```

Comment:
When doing arithmetic with $(()) you don't need to put a sigil before variables

Reply:
Sorry?

Reply:
`$(( minor + 1 )) ` is the same as `$(( $minor + 1 ))`

Inside an arithmetic expansion you don't need to prefix variables with sigil ($) because all non-numbers are treated as variables